collectors/charts.d.plugin/apcupsd/metadata.yaml

```diff
@@ -0,0 +1,66 @@
+  - metrics:
+      module:
+        name: apcupsd
```

Comment:
Why this structure? It doesn't reflect the one we have in netdata/go.d/plugin/modules/*.